assets/css/_diversion_page.scss

```diff
@@ -89,7 +93,7 @@
       }
     }
 
-    .l-diversion-page-panel__header .c-diversion-panel__back-button {
+    .l-diversion-page-panel__header .c-diversion-panel__outline-button--back {
```

Comment:
question: why change the name of this button? IMO the "outline" info doesn't seem necessary?

Reply:
You're totally right. I renamed at one point, assuming I'd use the `--back` as an extension of outline button, as in [these lines](https://github.com/mbta/skate/pull/2852/files#diff-a84a6c4e5f6eb9053ec860f02bebc5f2e4b0e544b079b3c814e92b6eaa6bea0aR169-R178), but not needed. (Plus, I see those might need tweaking as well)